### PR TITLE
ci: fix deprecation notice

### PR DESCRIPTION
**Pull request by SamuelAlev**
Screenshot 2023-08-30 at 14 15 58

Also:
- Cleaned the prefixes for pr title validation
- Removed automerge on renovate

### CODE REVIEW

File: .github/workflows/continuous-deployment.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ jobs:
 
             - name: Get pnpm store directory
               id: pnpm-cache
-              run: echo "::set-output name=pnpm_cache_dir::$(pnpm store path)"
+              run: echo "pnpm_cache_dir=$(pnpm store path)" >> $GITHUB_OUTPUT
```
Comment (ryancarville) [inline .github/workflows/continuous-deployment.yml:38]: Doesn't the `$GITHUB_OUTPUT` need to be wrapped in `"`? Looks like this one and `$GITHUB_ENV` use them.  
See doc here:
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter

Comment (SamuelAlev) [reply .github/workflows/continuous-deployment.yml:38]: Works the same, the data is still passed https://github.com/Frontify/fondue/actions/runs/6025904700/job/16347675219#step:6:3

**Review by ryancarville — APPROVED** [review]
LGTM 👍

File: .github/semantic.yml [hunk 2]
```diff
@@ -1,18 +1,14 @@
...
+    - docs
+    - refactor
+    - perf
+    - test
+    - ci
+    - chore
+    - release
+    - deps
+    - style
+    - ci
```
Comment (imoutaharik) [inline .github/semantic.yml:14]: @SamuelAlev `ci` prefix is not recognized for the PR title validation. We could add the same types on `pull-request-title.yml`

Comment (SamuelAlev) [reply .github/semantic.yml:12]: Fixed

**Review by imoutaharik — APPROVED** [review]

**Pull request merged**